play/src/main/scala/org/lyranthe/prometheus/client/play/filters/PrometheusFilter.scala

```diff
@@ -0,0 +1,57 @@
+package org.lyranthe.prometheus.client.play.filters
+
+import org.lyranthe.prometheus.client.{play => _, _}
+import org.lyranthe.prometheus.client.histogram.LabelledHistogram
```

Comment:
This looks to be too important a part of the API to be stuck in the `histogram` package - will move it to the base package (nothing required in this PR).